administration/src/application/components/ApplyController.tsx

```diff
@@ -69,7 +89,10 @@ const ApplyController = () => {
   }
 
   const submit = () => {
-    const validationResult = ApplicationForm.validate(state)
+    if (!regions) {
+      return
+    }
```

Comment:
```suggestion
```
I think, if you move the definition of `submit` below the loading/error/data-presence checks, we should not need this.